package.json

```diff
@@ -1,20 +1,53 @@
 {
   "name": "css-frameworks-ca",
-  "version": "1.0.0",
+  "version": "1.1.0",
   "description": "",
   "main": "index.js",
+  "type": "module",
   "scripts": {
+    "test": "npm run test-unit && npm run test-e2e",
+    "test-unit": "jest",
+    "test-e2e": "cypress open",
     "build": "sass src/scss:dist/css",
-    "start": "sass --watch src/scss:dist/css & live-server"
+    "start": "sass --watch src/scss:dist/css & live-server",
+    "format": "prettier -w src/js/**/*.js",
+    "lint": "eslint src/js/**/*.js",
+    "prepare": "husky install"
   },
   "keywords": [],
   "author": "Noroff",
   "license": "ISC",
   "devDependencies": {
+    "@babel/core": "^7.19.3",
+    "@babel/preset-env": "^7.19.4",
+    "cypress": "^10.11.0",
+    "eslint": "^8.32.0",
+    "eslint-plugin-cypress": "^2.12.1",
+    "eslint-plugin-jest": "^27.2.1",
+    "husky": "^8.0.3",
+    "jest": "^29.2.0",
+    "lint-staged": "^13.1.0",
     "live-server": "^1.2.2",
+    "prettier": "^2.8.3",
     "sass": "^1.54.8"
   },
   "dependencies": {
+    "bootstrap": "^5.2.3",
```

Comment:
Bootstrap is a peer dependency of bootstrap-dark-5, so this isn't really necessary

Reply:
I left this untouched. Is the solution to simply ```npm uninstall bootstrap``` ?

Reply:
Yep, but it's fine - a minor change